### ISSUE TITLE
Save Plugin as Folder only works for non-shadow tiddlers

### ISSUE THREAD

**Issue by mklauber** [OokTech/TW5-Bob#69]
Got a strange issue.  If I've got a plugin, and I go to Settings -> Bob Settings -> Manage Plugins, select it in the drop down menu, and click "Save Plugin as Folder" it doesn't save the contents of shadow tiddlers (e.g. the non overridden files in plugin).  It will create the file with the correct name, and it will display the "saved file ...." message, but only plugin tiddlers that have been edited and are no longer shadow tiddlers have any contents.  The rest are completely empty files.  I'm using BobExe for Linux 64 bit, version 1.1.0.  

---

I am running (check any that apply, put an x inside the [ ] to check a box, like this: [x]):

- [ ] Windows
- [ ] OSX
- [x] Linux
- [ ] Other

and using

- [ ] The nodejs version
- [x] The single file executable

Before posting I read [issue guidelines](./IssueGuidelines.md) and:

- [x] I am using the newest version
- [x] The answer to my question isn't listed in the documentation or this isn't
  a question
- [x] This is not a duplicate issue
- [ x I have not done anything that required me to set acceptance to
  `I Will Not Get Tech Support For This`

**Comment by mklauber** [comment]
I'm guessing the plugin isn't actually loaded on the server side, and thus the shadow tiddlers aren't in the `$tw.Bob.Wikis[data.wiki].wiki` object, but that's just a guess.

**Comment by inmysocks** [comment]
There is some misunderstanding here, but this is a strange bug.

The save plugin as folder should only save the packed plugin tiddler by splitting out the shadow tiddlers that make it up, it shouldn't look at anything other than the plugin tiddler itself.

So if you want to save a plugin and the plugin tiddler is $:/plugins/author/pluginname it only sends that one tiddler back to the server where it is split into its components and each tiddler is saved in the plugin folder.

I haven't used this a lot so I am not surprised that there are bugs, but this is unexpected.

**Comment by mklauber** [comment]
Fair enough.  Is there any way to hook up a debugger to the linux single file executable, or do I need node to debug the server?

**Comment by mklauber** [comment]
Finally found the time to track this down:  #74
